### DISCUSSION TITLE
Table lock on Select statement

### DISCUSSION THREAD

**Discussion by revigit21** [dbeaver/dbeaver#14305] [Q&A]
#### System information: 
- 
- DBeaver version :21.2.1
 
Helloo dbeavers, 
Currently no Test or Dev servers to access postgresSQL using Dbeaver Community version , so using Prod server for everything. Querying a table puts a lock on the table. Problem is nightly update jobs are stuck due to this issue as there are offshore folks working during that time. Is there a way to avoid locks on select statements? If we switch to Enterprise edition can this problem be avoided?

**Comment by Matvey16** [comment]
@revigit21 Have you tried using smart commit mode? It will open a transactions for any data modifying query, but auto-commits any select statement
